docs/data/query.md

```diff
@@ -185,6 +185,16 @@ For example, to connect to your Viam organization's MongoDB Atlas instance and q
      [ { numStanding: 215 } ]
      ```
 
+{{< alert title="Tip" color="tip" >}}
+If you use a data field that is named the same as a [reserved SQL keyword](https://en.wikipedia.org/wiki/List_of_SQL_reserved_words), such as `value` or `position`, you must escape that field name in your query using backticks ( <file>\`</file> ).
+For example, to query against a data field named `value` in the `readings` collection, you would use:
+
+```mongodb {class="command-line" data-prompt=">"}
+select sensorData.`value` from readings
```

Comment:
Nit, but I think we should have this say 
```sql
data.`value`
```
since there's no such field `sensorData` in the `readings` collection (`sensorData` is actually the name of the database containing the `readings` collection/table). But there is a field called `data`, that often has a subfield called `value` (depending on which components you've captured data from).